.github/workflows/image-publish.yml

```diff
@@ -77,8 +76,24 @@ jobs:
           SHORT_SHA1=$(git rev-parse --short=7 HEAD)
           if [[ ${{ env.BRANCH_NAME }} == main ]]; then
```

Comment:
If you're setting BRANCH_NAME to only equal `main` instead of `refs/heads/main` then your if-check on line 50 is going to fail because you removed `refs/heads/` on line 49